### PR TITLE
Add paging support to the Community Blog extractor

### PR DESCRIPTION
**Pull request by pento**
## Description

The community blog extractor currently exports the first 500 posts, categories, and tags.

This PR adds paging support, allowing it to export all data.

Fixes the last item in #6.

## How has this been tested?

- Tested in browser, reducing the page size to something more testable.
- Unit tests added.

### CODE REVIEW

**Comment by pento** [comment]
@akirk I wasn't quite sure of the structure you had in mind for `communities-blog-app/test/index.js`, so I ended up implementing these tests separately. There's a lot of duplicate code, however, did you have thoughts on reducing that?

File: source/services/wix/extractors/communities-blog-app/test/index.js [hunk 1]
```diff
@@ -14,15 +14,18 @@ const data = [
...
 		extracted: {},
 	},
 ];
 data[ 0 ].extracted.posts = data[ 0 ].posts;
 data[ 0 ].extracted.authors = data[ 0 ].authors;
 data[ 0 ].extracted.categories = data[ 0 ].categories;
-data[ 0 ].extracted.tags = data[ 0 ].tags;
+data[ 0 ].extracted.tags = data[ 0 ].tags.tags;
 
 test.each( data )( 'extract posts', async ( testData ) => {
```
Comment (akirk) [inline source/services/wix/extractors/communities-blog-app/test/index.js]: So my idea was to use multiple data arrays and iterate over them using `test.each()` and the comparison would happen against the extracted property.

**Review by akirk — APPROVED** [review]
I think your test works fine and I don't mind the duplication for now. I guess it could be refactored but that will likely make it less clear.

I'd rather actually get rid of the `test.each()` since it seems to have been unpractical for your use case.

**Comment by pento** [comment]
Thanks for the context, @akirk. I've moved the existing test to use the same format as the new tests. There's certainly room to reduce code duplication here, but it's not terrible, and I agree with your assessment that it would likely make the testing less clear.

**Pull request merged**